native/src/constants/contentDirection.ts

```diff
@@ -9,3 +9,15 @@ export const contentDirection = (contentLanguage: string): 'row' | 'row-reverse'
   isContentDirectionReversalRequired(contentLanguage) ? 'row-reverse' : 'row'
 export const contentAlignment = (contentLanguage: string): 'right' | 'left' =>
   isContentDirectionReversalRequired(contentLanguage) ? 'right' : 'left'
+export const isRTLText = (text: string): boolean => {
+  const rtlCount = (text.match(/[\u0591-\u07FF\uFB1D-\uFDFD\uFE70-\uFEFC]/g) || []).length
+  // eslint-disable-next-line no-misleading-character-class
+  const ltrCount = (
+    text.match(
+      // eslint-disable-next-line no-misleading-character-class
+      /[A-Za-z\u00C0-\u00C0\u00D8-\u00F6\u00F8-\u02B8\u0300-\u0590\u0800-\u1FFF\u2C00-\uFB1C\uFDFE-\uFE6F\uFEFD-\uFFFF]/g
```

Comment:
Not sure if makes sense to add some comments explaining the regexs.

Reply:
or at least where its from

Reply:
well okay i mean this rule checks for unicode chars of ltr languages and a-z but i can add a comment